pkg/api/authn.go

```diff
@@ -210,18 +217,20 @@ func basicAuthHandler(ctlr *Controller) mux.MiddlewareFunc {
 			if ok {
 				if err := bcrypt.CompareHashAndPassword([]byte(passphraseHash), []byte(passphrase)); err == nil {
 					// Process request
-					next.ServeHTTP(response, request)
+					ctx := getReqContextWithAuthorization(username, []string{}, request)
```

Comment:
We should also set the local groups (for HTTPPassword users) from the config file here, right?

Reply:
Only the ldap groups are kept on the context, the local groups are still kept as before. The getUserGroups function, which is in authz.go, does take both into consideration.

Reply:
Why not add all groups in the context?

Reply:
I have added them